pages/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
+permalink: /FAQ/
+nav_page: whatWeDo
+nav_page_specific: faq
+---
+
+# Frequently Asked Questions
+
+<hr class="bg-primary"/>
+
+Have a question about AB Tech, getting involved, or hiring us? We have answers (well, most of the time). Don't see your question below? Please email <a href = "mailto: abtech@andrew.cmu.edu">abtech@andrew.cmu.edu</a> with any questions you may have.
+
+- <a href="#General">General</a>
+- <a href="#Getting Involved">Getting Involved</a>
+- <a href="#Requesting AB Tech">Requesting AB Tech</a>
+
+<h2>
+ <a id="General">General</a>
+</h2>
+
+##### What is AB Tech? How do we fit into the Activities Board?
+AB Tech (short for Activities Board Technical Committee) is Carnegie Mellon’s production organization charged wtih providing the technical needs for university-sponsored events. We are one of the committees under the Activities Board (AB) along with Concerts, Events, Films, Publicity and Speakers; however, we are governed separately from the rest of the AB organization.
+
+##### What exactly does AB Tech do?
+Every year, CMU hosts many events on campus, such as dance shows, concerts, and competitions. These events often require technical support to run smoothly, and that is where AB Tech steps in. We provide the lighting, sound and backstage management needed to make the event a success. Some of our events are as small as providing speakers for a gathering, while others are as large as rigging new structures to hang dozens of lights for a concert. 
+
+##### Where is AB Tech located?
+Our main location is the techroom at the Cohon University Center. To get to techroom from ABP, follow the below directions:
+<ol>
+ <li>From the stairwell, head towards the Gym/Studio A/Keeler. You should pass the glass sculpture on the left.</li>
+ <li>Take a right and follow the hallway up the ramp.</li>
+ <li>Enter the double doors to your left at the top of the ramp.</li>
+</ol>
```

Comment:
It should be possible to do this without the `ol` and `li` tags. Try doing it like:
```markdown
1. From the stairwell, head towards the Gym/Studio A/Keeler. You should pass the glass sculpture on the left.
2. Take a right and follow the hallway up the ramp.
```